### PR TITLE
[Feature] Add divider, dividerStlye props

### PR DESCRIPTION
**Pull request by sk97kang**
## Description

Add divider, dividerStyle props

image

## Related Issues

#8 

## Tests

X

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [X] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [X] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [X] Run `yarn lint`
- [X] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/Card/index.tsx [hunk 1]
```diff
@@ -21,22 +21,39 @@ const StlyedImage = styled.Image`
   height: 100px;
 `;
 
+const Divider = styled.View`
+  margin: 5px 0px;
+  height: 0.7px;
+  background-color: lightgray;
```
Comment (greatSumini) [inline main/Card/index.tsx]: just question - doesn't it need any width style like (width : 100%;)?

File: main/Card/index.tsx [hunk 2]
```diff
@@ -21,22 +21,39 @@ const StlyedImage = styled.Image`
...
+    contentsStyle,
+    divider,
+    dividerStyle,
+  } = props;
   return (
     <Container style={[containerStyle]}>
       {image && <StlyedImage source={image} style={[imageStyle]} />}
       {children && (
         <ContentsContainer style={[contentsStyle]}>
+          {divider && <Divider style={[dividerStyle]} />}
```
Comment (greatSumini) [inline main/Card/index.tsx]:
```suggestion
          {title !== undefined && children && <Divider style={[dividerStyle]} />}
```
i think it's better to replace divider to title && children

File: main/Card/index.tsx (hunk 1 above)
Comment (sk97kang) [reply main/Card/index.tsx]: It's basically the whole thing.

File: main/Card/index.tsx (hunk 2 above)
Comment (sk97kang) [reply main/Card/index.tsx]: I changed

**Pull request merged**